src/components/accordion/accordion-header.tsx

```diff
@@ -0,0 +1,84 @@
+import styled, { css } from 'styled-components';
+import { TextStyles } from '../typography/typography-v2';
+import { ellipsis } from 'polished';
+import * as ReachAccordion from '@reach/accordion';
+import { theme } from '../layout/theme';
+import ChevronSVG from './chevron.svg';
+import { useAccordionItemContext } from '@reach/accordion';
+import React from 'react';
+
+interface AccordionHeaderProps {
+  children: React.ReactNode;
+}
+
+interface AccordionChevronProps {
+  open: boolean;
+  className?: string;
+}
+
+const AccordionTitleHeadline = styled.span`
+  ${TextStyles.headlineXS}
+  ${ellipsis()}
```

Comment:
this is coming from polish. We already have it in the code base so I replaced my manual text ellipsis code with this shortcut. See https://polished.js.org/docs/#ellipsis